.rubocop.yml

```diff
@@ -1327,8 +1327,6 @@ Rails/RedundantForeignKey:
   Enabled: true
 Rails/RedundantReceiverInWithOptions:
   Enabled: true
-Rails/ReflectionClassName:
```

Comment:
This one was giving me shit about using a string literal instead of calling a method like we do in `Aggregatable`. I could've just excluded that file but I felt like it was kind of an obscure issue anyway and might as well just cut the red tape.